scripts/portstat

```diff
@@ -210,10 +211,12 @@ class Portstat(object):
             Get the port speed
         """
         # Get speed from APPL_DB
-        full_table_id = PORT_STATUS_TABLE_PREFIX + port_name
         for ns in self.multi_asic.get_ns_list_based_on_options():
             self.db = multi_asic.connect_to_all_dbs_for_ns(ns)
-            speed = self.db.get(self.db.APPL_DB, full_table_id, PORT_SPEED_FIELD)
+            full_table_id = PORT_STATE_TABLE_PREFIX + port_name
```

Comment:
full_table_id is not required to be inside the for loop as its not changing. can you move this outside the for loop?